src/gluonts/model/seq2seq/_forking_estimator.py

```diff
@@ -363,51 +380,92 @@ def create_transformation(self) -> Transformation:
         return Chain(chain)
 
     def create_training_network(self) -> ForkingSeq2SeqNetworkBase:
-        return ForkingSeq2SeqTrainingNetwork(
-            encoder=self.encoder,
-            enc2dec=FutureFeatIntegratorEnc2Dec(),
-            decoder=self.decoder,
-            quantile_output=self.quantile_output,
-            context_length=self.context_length,
-            cardinality=self.cardinality,
-            embedding_dimension=self.embedding_dimension,
-            scaling=self.scaling,
-            scaling_decoder_dynamic_feature=self.scaling_decoder_dynamic_feature,
-            dtype=self.dtype,
-        )
+        if self.sampling:
```

Comment:
Following up from my other comment, this check here would be

```suggestion
        if self.distr_output is not None:
```